examples/additively_separable_scalar_field3_example.cc

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+//! [complete]
+#include <iostream>
+#include <ignition/math/AdditivelySeparableScalarField3.hh>
+#include <ignition/math/Polynomial3.hh>
+
+int main(int argc, char **argv)
+{
+  const double k = 1.;
+  const ignition::math::Polynomial3d px(
+      ignition::math::Vector4d(0., 1., 0., 1.));
+  const ignition::math::Polynomial3d qy(
+      ignition::math::Vector4d(1., 0., 1., 0.));
+  const ignition::math::Polynomial3d rz(
+      ignition::math::Vector4d(1., 0., 0., -1.));
+  using AdditivelySeparableScalarField3dT =
+      ignition::math::AdditivelySeparableScalarField3d<
+        ignition::math::Polynomial3d>;
+  const AdditivelySeparableScalarField3dT F(k, px, qy, rz);
```

Comment:
Style nit: variables use `camelBack`, and it's easier to read and find if it's more than one letter. So maybe:

```suggestion
  const AdditivelySeparableScalarField3dT field(k, px, qy, rz);
```

I see the same pattern on the tests.

Reply:
Fixed in https://github.com/ignitionrobotics/ign-math/pull/397/commits/ff97d1816923331a65f76a57ce8aea27a66ab8d0.